configuration-baselines/.htaccess-joomla

```diff
@@ -186,6 +186,14 @@ RewriteRule ^(htaccess\.txt|configuration\.php-dist|php\.ini|\.user\.ini)$ - [F]
 	# Strict-Transport-Security
 	Header always set Strict-Transport-Security "max-age=31536000; includeSubDomains"
 </IfModule>
+
+## Since Joomla 3.9.21
```

Comment:
Why should that been added here? You have history via git right?

Reply:
This is our baseline for new sites.

Reply:
Ok up to you